tests/benchmark/run.py

```diff
@@ -25,7 +25,7 @@ def subtract(after_dict, before_dict):
     return {key: after_dict[key] - before_dict.get(key, 0) for key in after_dict}
 
 
-def profile(func, *args, **kwargs):
+def profile(func, *args, **kwargs):  # noqa: C901
```

Comment:
Added an ignore comment here as the Code Complexity here is better for readability, logical flow and its in our benchmark script.